homeassistant/components/airvisual/strings.json

```diff
@@ -18,6 +18,13 @@
           "password": "[%key:common::config_flow::data::password%]"
         }
       },
+      "reauth_confirm": {
+        "title": "Re-authenticate AirVisual",
+        "description": "Input a valid AirVisual Cloud API key.",
```

Comment:
I don't think that this description adds anything. All API keys should be valid?

```suggestion
```